src/Commands/CreateCommand.php

```diff
@@ -59,9 +60,12 @@ class CreateCommand extends TerminusCommand implements SiteAwareInterface
      *     Creates a backup of all elements in all site environments associated with any organization of which the currently logged-in is a member.
      * @usage terminus backup-all:create --name=<regex>
      *     Creates a backup of all elements in all site environments with a name that matches <regex>.
+     * @usage terminus backup-all:create --async
+     *     Creates a backup of all elements but does not wait for backup to finish request is done asynchronous.
```

Comment:
`Creates a backup of all elements but does not wait for the backup to finish since the request is asynchronous.` ?